back/public/javascripts/file.js

```diff
@@ -197,24 +215,11 @@ frostbite.controller('FileCtrl', ['$scope','$http', '$q', function ($scope, $htt
                 console.log(lineItem.line);
                 console.log(lineItem.totalCount)
                 var color = 'background-color:';
-                //Currently modifying color after page load, hence the modifying of style element
-                //If we know importance on page load, revert to modifying css class name
-
-                if (lineItem.totalCount > 2) {
-                    color += '#F37E91';
-                }
-                else if (lineItem.totalCount > 1) {
-                    color += '#F9A2B0';
-                }
-                else if (lineItem.totalCount > 0) {
-                    color += '#FDDAE0';
-                }
 
+                color += colors[Math.floor(lineItem.totalCount/maxLineCount * 10)];
```

Comment:
because we have 10 colours currently, it should be colors.length though in case we change that